tests/Neo.Network.RPC.Tests/RpcTestCases.json

```diff
@@ -427,7 +427,7 @@
                   }
                 ],
                 "offset": 0,
-                "returnType": "Signature"
+                "returnType": "Any"
```

Comment:
@shargon  @erikzhang  Do you know why did it change here? It seems  we haven't changed `event`.

Reply:
I am not sure...

Reply:
Why `event` has a return type?

Reply:
It should be removed.

Reply:
See https://github.com/neo-project/neo/pull/1733